src/frontends/lean/decl_cmds.cpp

```diff
@@ -139,6 +139,7 @@ static environment declare_var(parser & p, environment env,
         if (meta.m_modifiers.m_is_protected)
             env = add_protected(env, full_n);
         env = ensure_decl_namespaces(env, full_n);
+        env = meta.m_attrs.apply(env, p.ios(), full_n);
         return env;
```

Comment:
This is a bug fix, right? 
We were ignoring attribute declarations in this case.

Reply:
Yes. The test "broken" by the fix used "@[simp] axiom" and did one more `simp` than should be necessary.